### PR TITLE
[SYCL][JM tests] fix syntax error for LIT

### PR DESCRIPTION
**Pull request by dkhaldi**

### CODE REVIEW

**Review by YuriPlyakhin — APPROVED** [review]
LGTM

**Comment by dkhaldi** [comment]
@intel/llvm-gatekeepers, the failure seems generic, I created an issue in https://github.com/intel/llvm/issues/11405.
If you agree, can you help merge?

File: sycl/test-e2e/Matrix/SG32/joint_matrix_bf16_fill_k_cache_unroll.cpp [hunk 1]
```diff
@@ -10,7 +10,7 @@
 // RUN: %if gpu %{ %{build} -mllvm -inline-threshold=5000 -o %t_gpu.out -DSYCL_EXT_ONEAPI_MATRIX_VERSION=4 -DMANUAL_UNROLL %}
 // RUN: %if gpu %{ %{run} %t_gpu.out %}
 
-// RUN: %if cpu %{ %{build} -mllvm -inline-threshold=5000 -o %t_cpu.out -DSYCL_EXT_ONEAPI_MATRIX_VERSION=4 -DMANUAL_UNROLL -DtM=16 -DtK=32 -DNCACHE1=32 -DKCACHE1=32
+// RUN: %if cpu %{ %{build} -mllvm -inline-threshold=5000 -o %t_cpu.out -DSYCL_EXT_ONEAPI_MATRIX_VERSION=4 -DMANUAL_UNROLL -DtM=16 -DtK=32 -DNCACHE1=32 -DKCACHE1=32 %}
 // RUN: %if cpu %{ %{run} %t_cpu.out %}
 
 // XFAIL: gpu
```
Comment (YuriPlyakhin) [inline sycl/test-e2e/Matrix/SG32/joint_matrix_bf16_fill_k_cache_unroll.cpp:16]: // RUN: %{build} -mllvm -inline-threshold=5000 -o %t_gpu.out -DSYCL_EXT_ONEAPI_MATRIX_VERSION=4 -DMANUAL_UNROLL
// RUN: %{build} -mllvm -inline-threshold=5000 -o %t_cpu.out -DSYCL_EXT_ONEAPI_MATRIX_VERSION=4 -DMANUAL_UNROLL -DtM=16 -DtK=32 -DNCACHE1=32 -DKCACHE1=32

// RUN: %if gpu %{ %{run} %t_gpu.out %}
// RUN: %if cpu %{ %{run} %t_cpu.out %}

Comment (YuriPlyakhin) [reply sycl/test-e2e/Matrix/SG32/joint_matrix_bf16_fill_k_cache_unroll.cpp:16]: Hi @dkhaldi , I checked that adding %} at the end is not enough to fix these tests.
if gpu/if cpu doesn't work for builds, so we should build for gpu and cpu unconditionally.
Then run either cpu or gpu, when condition satisfied.
Also, please, remote xfail:gpu.
See the code I inserted above

Comment (YuriPlyakhin) [reply sycl/test-e2e/Matrix/SG32/joint_matrix_bf16_fill_k_cache_unroll.cpp:16]: The same should be applied for all 4 tests.

**Review by YuriPlyakhin — CHANGES_REQUESTED** [review]

**Review by YuriPlyakhin — APPROVED** [review]
LGTM

**Comment by YuriPlyakhin** [comment]
@intel/llvm-gatekeepers , please, merge, all green

**Pull request merged**